frontend/src/app/modules/ifc_models/bcf/container/bcf-container.component.ts

```diff
@@ -13,9 +13,6 @@ import {untilComponentDestroyed} from "ng2-rx-componentdestroyed";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/732e2fbbcf0087ed091db460fcf58f272bd92383/frontend/src/app/modules/ifc_models/bcf/container/bcf-container.component.ts#L13"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>